packages/grid/README.md

```diff
@@ -0,0 +1,59 @@
+# Grid component for the Undercoat CSS Framework
+
+## Grid system
+
+The grid is based on 24 columns (tracks) by default. It's like 12 columns but more flexible since it can be split into 2, 3, 4 and even 6, 8 and 12 equal parts! But you can configure the number of columns following your requirements.
+
+The default grid settings are stored in CSS custom properties and are below (px units):
+```CSS
+  --grid-columns-number: 24; /* columns number */
+  --grid-columns-min: 13;    /* column (track) min width */
```

Comment:
The name of the variable does not reflect the fact that it contains the width, not the number.

Reply:
Fixed in 26a64d14115482791700894574a1859be37c32b0.